ortools/linear_solver/sirius_interface.cc

```diff
@@ -273,6 +281,34 @@ SiriusInterface::SiriusInterface(MPSolver* const solver, bool mip)
   // SRS_OBJ_MINIMIZE));
 }
 
+void SiriusInterface::SetStartingLpBasis(
+    const std::vector<MPSolver::BasisStatus>& variable_statuses,
+    const std::vector<MPSolver::BasisStatus>& constraint_statuses) {
+  has_basis = true;
```

Comment:
This assignment must be placed after the 2 checks on sizes, at l. 302

Otherwise we might incorrectly assume we have a basis when there isn't because of size mismatch here.